src/vs/workbench/parts/files/browser/fileActions.ts

```diff
@@ -560,7 +560,7 @@ export abstract class BaseGlobalNewAction extends Action {
 }
 
 /* Create new file from anywhere: Open untitled */
-export class GlobalNewFileAction extends Action {
+export class GlobalNewUntitledFileAction extends Action {
 	public static ID = 'workbench.action.files.newUntitledFile';
 	public static LABEL = nls.localize('newFile', "New File");
```

Comment:
That would definitely be better! How can I do that as probably it should become `('newUntitledFile', "New Untitled File")` and those translations should be adjusted accordingly right? Or are the translations done separately before release?